docs/reference/sql/functions/conditional.asciidoc

```diff
@@ -0,0 +1,43 @@
+[role="xpack"]
+[testenv="basic"]
+[[sql-functions-conditional]]
+=== Conditional Functions
+
+Functions that return one of their arguments by evaluating in an if-else manner.
+
+[[sql-functions-conditional-coalesce]]
+==== `COALESCE`
+
+.Synopsis
+[source, sql]
+----
+COALESCE ( expression<1>, expression<2>, ... )
+----
+
```

Comment:
Enumerating the parameters should happen in a section called `Input`. Also, there has to be an `Output` section. Something like:

```
==== `COALESCE`

.Synopsis:
[source, sql]
--------------------------------------------------
COALESCE ( expression<1>, expression<2>, ... )
--------------------------------------------------

*Input*:

<1> 1st expression
<2> 2nd expression
....

*Output*: one expression or `null`

.Description:

Returns the first of its arguments that is not null.
If all arguments are null, then it returns `null`.
```

Reply:
Also, I would add in the description (to enforce the `...` in the parameters list), that the function can take an arbitrary number of arguments.

Reply:
The statement about arbitrary number of arguments is there. Please, ignore my comment.